openshift_images/configuring-samples-operator.adoc

```diff
@@ -23,15 +23,9 @@ The Cluster Samples Operator, which operates in the `openshift` namespace, insta
 endif::openshift-rosa,openshift-dedicated[]
 
 [IMPORTANT]
-.Cluster Samples Operator is being downsized  
+.The Cluster Samples Operator is being deprecated  
```

Comment:
I like the heading in the admonition. Not sure I've seen it before, but it works to attract attention and explain the issue immediately .

Reply:
@mburke5678 Thank you! Glad you like it :)